api/expiring_certificates_service.go

```diff
@@ -36,11 +44,17 @@ func (a Api) ListExpiringCertificates(expiresWithin string) ([]ExpiringCertifica
 		return nil, err
 	}
 
-	var expiringCertificatesResponse ExpiringCertificatesResponse
-	if err := json.NewDecoder(resp.Body).Decode(&expiringCertificatesResponse); err != nil {
+	var certificatesResponse ExpiringCertificatesResponse
+	if err := json.NewDecoder(resp.Body).Decode(&certificatesResponse); err != nil {
 		return nil, err
 	}
 	defer resp.Body.Close()
 
-	return expiringCertificatesResponse.Certificates, nil
+	return certificatesResponse.Certificates, nil
+}
+
+// ListExpiringCertificates retrieves certificates that expire within the specified timeframe.
+// This method maintains backward compatibility with existing code.
+func (a Api) ListExpiringCertificates(expiresWithin string) ([]ExpiringCertificate, error) {
+	return a.ListCertificates(expiresWithin)
 }
```

Comment:
Are there callers of this function outside of the `om` codebase? If not, I wouldn't keep this passthrough method anymore